src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -83,12 +83,13 @@ export class GraphStyles {
             const rate = Number(ele.data('rate'));
             if (rate > 0) {
               const pErr = ele.data('percentErr') ? Number(ele.data('percentErr')) : 0;
-              content = pErr > 0 ? rate.toFixed(2) + ', ' + pErr.toFixed(1) + '%' : rate.toFixed(2);
+              const rateStr = rate < 1 ? rate.toFixed(3) : rate.toFixed(2);
+              content = pErr > 0 ? rateStr + ', ' + pErr.toFixed(1) + '%' : rateStr;
             }
           } else if (ele.data('tcpSentRate')) {
             const rate = Number(ele.data('tcpSentRate'));
             if (rate > 0) {
-              content = `${rate.toFixed(2)}`;
+              content = `${rate.toFixed(3)}`;
```

Comment:
Do we need to update the side panel with the same value?

Reply:
Done!